model-api/src/commonTest/kotlin/org/modelix/model/api/BuiltinLanguagesTest.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023-2024.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.modelix.model.api
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class BuiltinLanguagesTest {
+
+    @Test
+    fun propertiesAreAdded() {
```

Comment:
Maybe something like "allPropertiesAreAvailableWithoutPriorAccess"?

Reply:
Done. Also added more detailed comments in the test code.